feature/sessions/src/main/java/io/github/droidkaigi/confsched2023/sessions/component/TimetableGridItem.kt

```diff
@@ -136,3 +153,19 @@ fun PreviewTimetableGridItem() {
         }
     }
 }
+
+@Preview(uiMode = Configuration.UI_MODE_NIGHT_NO)
+@Preview(uiMode = Configuration.UI_MODE_NIGHT_YES)
+@Composable
+private fun PreviewTimetableGridItem(
```

Comment:
Private modifier was added because of a Lint error.

Reply:
May I ask what lint error you see? 👀

Reply:
Screenshots and text are here. 💁‍♂️

> Composables annotated with @Preview that are used only for previewing the UI should not be public.
![image](https://github.com/DroidKaigi/conference-app-2023/assets/13657682/fd6b60c8-171f-441a-9835-90c422439c0d)

Reply:
I'm wandering this will prevent us to check Preview in pull requests 👀

Reply:
For example, can we use an internal modifier instead of private?

Reply:
I'll check issues for the slack compose lint or showkase but it will be late. So I was wondering if you could this 🙇

Reply:
Now we can use custom preview annotations,  this would work now?